tests/service/privacy_request/request_runner_service_test.py

```diff
@@ -129,7 +131,7 @@ def test_create_and_process_access_request(
 
     policy.delete(db=db)
     pr.delete(db=db)
-    db.expunge_all()
```

Comment:
This line was causing downstream failures in the test teardown where FKs also needed to be cleaned up (specifically the `ConnectionConfig` -> `DatasetConfig` relation). After talking to @pattisdr I discovered this line was here as a forcing function to ensure that `pr` is no longer in the database session when the execution logs are checked for their `.privacy_request_id` attribute, a step to ensure that value is not cleared when `pr` is deleted.

I've opted to manually check whether `pr` is in the session in the line below, as this performs the same function.